docs/pages/management/admin/self-signed-certs.mdx

```diff
@@ -0,0 +1,117 @@
+running it - via `teleport` directly from the cli, or via helm chart, or via systemd:
+<Tabs>
+
+    <TabItem label="Running teleport directly">
+    When running `teleport` from the command line, pass the `--insecure` flag to disable
+    TLS certificate validation. For example:
+    ```sh
+    $ teleport start -c /etc/teleport.yaml --insecure
+    $ teleport app start -c /etc/teleport.yaml --insecure
+    $ teleport db start -c /etc/teleport.yaml --insecure
+    ```
+    Without the `--insecure` flag, you will see an error message that looks like
+    `x509: “tele.example.com” certificate is not trusted`.
+    </TabItem>
+
+    <TabItem label="Running teleport via helm chart">
+    For deploying a cluster, set
+    [extraArgs](../../reference/helm-reference/teleport-cluster.mdx#extraargs)
+    to include the extra argument: `--insecure`.
+    
+    
+    For a Helm Kube Agent, set the 
+    [insecureSkipProxyTLSVerify
+    ](../../reference/helm-reference/teleport-kube-agent.mdx#insecureskipproxytlsverify)
+    flag to `true`.
+    </TabItem>
+
+    <TabItem label="Running teleport via systemd">
+    Edit the `systemd` Teleport unit file to include `--insecure` in the `ExecStart` line, for example:
+    
+    `ExecStart=/usr/local/bin/teleport start --pid-file=/run/teleport.pid --insecure`
+    
+    You also need to run `sudo systemctl daemon-reload` and then `sudo systemctl restart teleport.service`
+    for your change to take effect.
+    
+    See [create a systemd service](./daemon.mdx#step-23-create-and-configure-a-systemd-service) for
+    more information.
+    </TabItem>
+
+</Tabs>
+
+## Self-Signed Certs: `tctl`
+When running `tctl` remotely via the Teleport Proxy Service, if the proxy is using self-signed certs,
+then `tctl` will not trust the certificate from the proxy. To disable certificate verification use
+`tctl --insecure`.
+`tctl` will determine how to connect to the auth service in a few ways:
+- loading configuration from the profile after logging in with `tsh`
+- loading from a config file passed as an argument: `tctl -c /etc/teleport.yaml`
+- passing the `--auth-server` flag directly, as in:
+  - `tctl --auth-server=tele.example.com:443` or
+  - `tctl --auth-server=tele.example.com:3080`
+
+If any of these methods tries to connect via the Teleport Proxy Service, and the proxy is using self-signed
+certs, then `tctl` will not trust the certificate from the proxy and you will get an error message about
+untrusted or invalid certificates, unless `--insecure` is also passed to `tctl`.
+
+## Self-Signed Certs: `tsh`
+When running `tsh`, you must specify the Teleport Proxy Service address for `tsh` to connect to.
+If the Teleport Proxy Service is using self-signed certs, then `tsh` will not trust the proxy's certificate.
+In order to use `tsh` in this case, you need to use the `--insecure` flag.
+
+## Configuring Certificates Properly
```

Comment:
I don't think we need this section, since this guide is explicitly intended for users who have already decided to use self-signed certificates in an insecure environment. If we do want to direct the user to instructions on setting up proper PKI with Teleport, we could, for example, link to https://goteleport.com/docs/deploy-a-cluster/open-source/

Reply:
ha, this is actually a case where I think inlining is warranted since the guide emphasizes the importance of setting up certs properly in prod. I don't feel strongly about it though, so I can replace the inline with a link here

Reply:
I'm not sure this guide is about the importance of setting up certs properly. To me, up until this section, it's explicitly focused on how to use self-signed certificates in non-production environments. If we want to reframe this guide to be about how the Proxy Service uses TLS in general, we could include this section, but we should also:
- move this to a different section of the docs, e.g., `/docs/architecture`
- change the title and introductory section to be about how the Proxy Service uses TLS in general
- make this more of a conceptual guide than a how-to guide

Reply:
This is related to the link discussion we had above, and I think in this case I'll just remove the inline guide and provide a link under "Further Reading" at the bottom of the page.